docs/how-to-guides/deploy-a-flow/deploy-using-docker.md

```diff
@@ -53,7 +53,7 @@ refer to [Setup connection for web-classifiction](https://github.com/microsoft/p
 Use the command below to build a flow as docker format app:
 
 ```bash
-pf flow build --source ../../flows/standard/web-classification --output build --format docker
+pf flow build --source ../../flows/standard/web-classification --output dist --format docker
```

Comment:
I think we might have more of this, pls give it double check

Reply:
checked and updated